bin/anthology/people.py

```diff
@@ -21,20 +21,29 @@
 class PersonName:
     first, last = "", ""
 
-    def __init__(self, first, last):
+    def __init__(self, first, last, variant: "PersonName" = None):
```

Comment:
Is it conceivable that there could be more than one variant?

Reply:
Yes, but nothing I'm doing now prevents expanding later.